app/views/newflow/login_signup/educator_signup_form.html.erb

```diff
@@ -137,6 +137,7 @@
             </label>
             <%= f.hidden_field :contract_1_id, value: contracts.first.id %>
             <%= f.hidden_field :contract_2_id, value: contracts.second.id %>
+            <%= f.hidden_field :role, value: :instructor %>
```

Comment:
FYI, this was fixed in https://github.com/openstax/accounts/pull/786 (which is currently blocked).  Sounds good to get it in now of course. Thanks!